Makefile

```diff
@@ -1,7 +1,7 @@
 CC ?= cc
 CFLAGS ?= -O2 -fPIC
 PREFIX ?= /usr/local
-VC ?= 0.1.17
+VC ?= e0c9a84
```

Comment:
why not using `git rev-parse --short HEAD` to get the short hash of the current local head

Reply:
vc is another repository

Reply:
true,but we can use `something like 
`git ls-remote https://github.com/vlang/vc.git --tags master | awk '{print $1;}'`

Reply:
Specifying a version can improve reproducibility.

Reply:
but the hash will change after this commit got marged

Reply:
but vc is another repository, so hash will not change

Reply:
ohh ,ok sry was a long day

Reply:
manually updating makefile every time vc is changed is not an option